repose-aggregator/components/filters/attribute-mapping-policy-validation-filter/build.gradle

```diff
@@ -0,0 +1,18 @@
+dependencies {
+    compile project(":repose-aggregator:commons:commons-utilities")
+    compile "org.scala-lang:scala-library"
+    compile "com.typesafe.scala-logging:scala-logging-slf4j_$scalaMajDotMin"
+    compile "com.rackspace.identity.components.attribute-mapper:mapper-core"
+    //todo: remove this once we can move the translation filter up to the same version of saxon
+    compile "net.sf.saxon:Saxon-EE:9.7.0-8"
+    compile "com.fasterxml.jackson.core:jackson-databind"
```

Comment:
Unfortunately, the Attribute Mapper library API forces us to deal with the Saxon, Jackson, and Java XML APIs as well. That is why these dependencies are here.